shared/src/commonMain/kotlin/org/jetbrains/kotlinconf/Model.kt

```diff
@@ -3,9 +3,22 @@ package org.jetbrains.kotlinconf
 import io.ktor.util.date.*
 import kotlinx.serialization.*
 import org.jetbrains.kotlinconf.utils.*
+import kotlin.jvm.JvmInline
 
 typealias GMTDateSerializable = @Serializable(GMTDateSerializer::class) GMTDate
 
+@Serializable
+@JvmInline
+value class SpeakerId(val id: String) {
+    override fun toString(): String = id
+}
+
+@Serializable
+@JvmInline
+value class SessionId(val id: String) {
+    override fun toString(): String = id
+}
```

Comment:
These new wrapper classes broke the `backend` module, the CI check is failing to compile

Reply:
Fixed in #214